libs/cimit-service/src/test/java/uk/gov/di/ipv/core/library/service/CiMitServiceTest.java

```diff
@@ -219,7 +219,7 @@ void getContraIndicatorsVC() throws Exception {
                 new String(request.getPayload().array(), StandardCharsets.UTF_8));
 
         assertEquals(
-                "ContraIndicators(contraIndicatorsMap={D01=ContraIndicator(code=D01, issuanceDate=2022-09-20T15:54:50.000Z, document=passport/GBR/824159121, txn=[abcdef], mitigation=[Mitigation(code=M01, mitigatingCredential=[MitigatingCredential(issuer=https://credential-issuer.example/, validFrom=2022-09-21T15:54:50.000Z, txn=ghij, id=urn:uuid:f81d4fae-7dec-11d0-a765-00a0c91e6bf6)])], incompleteMitigation=[Mitigation(code=M02, mitigatingCredential=[MitigatingCredential(issuer=https://another-credential-issuer.example/, validFrom=2022-09-22T15:54:50.000Z, txn=cdeef, id=urn:uuid:f5c9ff40-1dcd-4a8b-bf92-9456047c132f)])])})",
+                "ContraIndicators(contraIndicatorsMap={D01=ContraIndicator(code=D01, issuers=null, issuanceDate=2022-09-20T15:54:50.000Z, document=passport/GBR/824159121, txn=[abcdef], mitigation=[Mitigation(code=M01, mitigatingCredential=[MitigatingCredential(issuer=https://credential-issuer.example/, validFrom=2022-09-21T15:54:50.000Z, txn=ghij, id=urn:uuid:f81d4fae-7dec-11d0-a765-00a0c91e6bf6)])], incompleteMitigation=[Mitigation(code=M02, mitigatingCredential=[MitigatingCredential(issuer=https://another-credential-issuer.example/, validFrom=2022-09-22T15:54:50.000Z, txn=cdeef, id=urn:uuid:f5c9ff40-1dcd-4a8b-bf92-9456047c132f)])])})",
```

Comment:
We should probably update the `SIGNED_CONTRA_INDICATOR_VC` with issuers, so this doesn't come back as null